database-crud/src/datasource.ts

```diff
@@ -5,7 +5,7 @@ const AppDataSource = new DataSource({
   type: "postgres",
   url: "postgres://postgres@localhost:5432/postgres",
   ssl: false,
-  synchronize: false,
+  synchronize: true,
```

Comment:
See if you can leave this as `false` and use the migrations to setup the database schema.
```suggestion
  synchronize: false,
```